syft/file/selection.go

```diff
@@ -0,0 +1,9 @@
+package file
+
+const (
+	NoFilesSelection    Selection = "no-files"
+	OwnedFilesSelection Selection = "owned-files"
```

Comment:
Does `owned-files` mean "files owned by a package?" (Out of context I thought it meant files owned by a group or user for a second.)

Reply:
> Does owned-files mean "files owned by a package?"

correct 👍

Reply:
renamed 👍